archway-1/gentx/gentx-bae46a52c009237992fbe3119210a3ce80d5ef6c.json

```diff
@@ -0,0 +1 @@
+{"body":{"messages":[{"@type":"/cosmos.staking.v1beta1.MsgCreateValidator","description":{"moniker":"Astake","identity":"44937E3DA9AA699A","website":"https://gatadao.zone","security_contact":"","details":"Decentralization and security focused community-owned PoS infrastructure with in-house sentry node architecture. Join GATA family"},"commission":{"rate":"0.010000000000000000","max_rate":"0.100000000000000000","max_change_rate":"0.100000000000000000"},"min_self_delegation":"1","delegator_address":"archway1fzncp86ne4qg2kkg9wc6vmmkgs9k42wurmrfxh","validator_address":"archwayvaloper1fzncp86ne4qg2kkg9wc6vmmkgs9k42wurl2tt7","pubkey":{"@type":"/cosmos.crypto.ed25519.PubKey","key":"mnIJjxEOIh+pdWsvLUpqx3CXXUiJTWnL5MZDxzVWgc0="},"value":{"denom":"aarch","amount":"9500000000000000000"}}],"memo":"bae46a52c009237992fbe3119210a3ce80d5ef6c@65.109.158.20:26656","timeout_height":"0","extension_options":[],"non_critical_extension_options":[]},"auth_info":{"signer_infos":[{"public_key":{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"AhpruXHpbjdIIpPnH2MdosO6S7AbfO4lgQMKmznluavq"},"mode_info":{"single":{"mode":"SIGN_MODE_DIRECT"}},"sequence":"0"}],"fee":{"amount":[{"denom":"aarch","amount":"180000000000000000"}],"gas_limit":"200000","payer":"","granter":""}},"signatures":["aImQJq6iPG1fqzXw4z9EUrP6bQgH7V+G6T62nRyLQFNFK3dDDKDEXV6br6jzgtYPMDYBsLZEuZgKqIeuP/0yJA=="]}
```

Comment:
Validation fails with an error:
```
Error: failed to get genesis app state from config: account archway1fzncp86ne4qg2kkg9wc6vmmkgs9k42wurmrfxh balance not in genesis state: 
```
@JoveCosmos please ensure gentx is submitted with a validator address provided to us

Reply:
Yes, sorry for that, updated.